<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="2" total="3" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-06-25T20:12:09 IST" name="Default suite" finished-at="2022-06-25T20:12:19 IST" duration-ms="10427">
    <groups>
    </groups>
    <test started-at="2022-06-25T20:12:09 IST" name="Default test" finished-at="2022-06-25T20:12:19 IST" duration-ms="10427">
      <class name="Testcase.LoginTestcase">
        <test-method is-config="true" signature="setup()[pri:0, instance:Testcase.LoginTestcase@65f095f8]" started-at="2022-06-25T20:12:09 IST" name="setup" finished-at="2022-06-25T20:12:18 IST" duration-ms="9028" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="VerifyTitletest()[pri:0, instance:Testcase.LoginTestcase@65f095f8]" started-at="2022-06-25T20:12:18 IST" name="VerifyTitletest" finished-at="2022-06-25T20:12:18 IST" duration-ms="107" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Kite - Zerodha's fast and elegant flagship trading platform] but found [1Kite - Zerodha's fast and elegant flagship trading platform]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Kite - Zerodha's fast and elegant flagship trading platform] but found [1Kite - Zerodha's fast and elegant flagship trading platform]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at Testcase.LoginTestcase.VerifyTitletest(LoginTestcase.java:24)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyTitletest -->
        <test-method is-config="true" signature="exit()[pri:0, instance:Testcase.LoginTestcase@65f095f8]" started-at="2022-06-25T20:12:18 IST" name="exit" finished-at="2022-06-25T20:12:19 IST" duration-ms="1206" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- exit -->
      </class> <!-- Testcase.LoginTestcase -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
